pymatgen/io/lobster/inputs.py

```diff
@@ -728,114 +731,120 @@ def standard_calculations_from_vasp_files(
             lobsterin_dict["skipGrossPopulation"] = True
             lobsterin_dict["skipMadelungEnergy"] = True
 
-        if option == "onlydos":
+        elif option == "onlydos":
             lobsterin_dict["skipcohp"] = True
             lobsterin_dict["skipcoop"] = True
             lobsterin_dict["skipPopulationAnalysis"] = True
             lobsterin_dict["skipGrossPopulation"] = True
-            # lobster-4.1.0
+            # LOBSTER-4.1.0
             lobsterin_dict["skipcobi"] = True
             lobsterin_dict["skipMadelungEnergy"] = True
 
-        if option == "onlyprojection":
+        elif option == "onlyprojection":
             lobsterin_dict["skipdos"] = True
             lobsterin_dict["skipcohp"] = True
             lobsterin_dict["skipcoop"] = True
             lobsterin_dict["skipPopulationAnalysis"] = True
             lobsterin_dict["skipGrossPopulation"] = True
             lobsterin_dict["saveProjectionToFile"] = True
-            # lobster-4.1.0
+            # LOBSTER-4.1.0
             lobsterin_dict["skipcobi"] = True
             lobsterin_dict["skipMadelungEnergy"] = True
 
-        if option == "onlycobi":
+        elif option == "onlycobi":
             lobsterin_dict["skipdos"] = True
             lobsterin_dict["skipcohp"] = True
             lobsterin_dict["skipPopulationAnalysis"] = True
             lobsterin_dict["skipGrossPopulation"] = True
-            # lobster-4.1.0
+            # LOBSTER-4.1.0
             lobsterin_dict["skipcobi"] = True
             lobsterin_dict["skipMadelungEnergy"] = True
 
-        if option == "onlymadelung":
+        elif option == "onlymadelung":
             lobsterin_dict["skipdos"] = True
             lobsterin_dict["skipcohp"] = True
             lobsterin_dict["skipcoop"] = True
             lobsterin_dict["skipPopulationAnalysis"] = True
             lobsterin_dict["skipGrossPopulation"] = True
             lobsterin_dict["saveProjectionToFile"] = True
-            # lobster-4.1.0
+            # LOBSTER-4.1.0
             lobsterin_dict["skipcobi"] = True
 
         incar = Incar.from_file(INCAR_input)
+
         if incar["ISMEAR"] == 0:
             lobsterin_dict["gaussianSmearingWidth"] = incar["SIGMA"]
+
         if incar["ISMEAR"] != 0 and option == "standard_with_fatband":
             raise ValueError("ISMEAR has to be 0 for a fatband calculation with Lobster")
+
         if dict_for_basis is not None:
-            # dict_for_basis={"Fe":'3p 3d 4s 4f', "C": '2s 2p'}
-            # will just insert this basis and not check with poscar
+            # dict_for_basis = {"Fe":"3p 3d 4s 4f", "C": "2s 2p"}
+            # Will just insert this basis and not check with poscar
             basis = [f"{key} {value}" for key, value in dict_for_basis.items()]
         elif POTCAR_input is not None:
-            # get basis from POTCAR
-            potcar_names = Lobsterin._get_potcar_symbols(POTCAR_input=POTCAR_input)
+            # Get basis from POTCAR
```

Comment:
```suggestion
            # Get basis functions from POTCAR
```

Reply:
Perhaps we need to change this for consistency as well: https://github.com/materialsproject/pymatgen/blob/2982968a44d0ff65392057488aee4871210bc847/pymatgen/io/lobster/inputs.py#L342-L348

Reply:
It might be a good idea to be language-wise concise here as well, to just prevent that people potentially confuse it with "basis set".